ministryofjustice/analytics-platform-control-panel-frontend
# bump node to (almost) latest LTS version

## What

Bump nodejs to LTS version of the week, 8.9.3 (there's no dockerfile tag for 8.9.4 yet...)

## How to review

1. check out, pull and build and up your dockers
2. see that the app runs
3. cry just a little bit
